gallery/gallery/tool/grind.dart

```diff
@@ -54,12 +54,37 @@ Future<void> l10n() async {
 @Task('Update code segments')
 Future<void> updateCodeSegments() async {
   final codeviewerPath =
-      path.join(Directory.current.parent.path, 'codeviewer_cli');
-  await pubGet(directory: codeviewerPath);
+      path.join(Directory.current.path, 'tool', 'codeviewer_cli');
 
-  Dart.run(path.join(codeviewerPath, 'bin', 'main.dart'));
-  final targetFilePath = path.join('lib', 'codeviewer', 'code_segments.dart');
-  await format(path: targetFilePath);
+  Dart.run(path.join(codeviewerPath, 'main.dart'));
+  final codeSegmentsPath = path.join('lib', 'codeviewer', 'code_segments.dart');
+  await format(path: codeSegmentsPath);
+}
+
+@Task('Verify and analyze dart files')
+Future<void> analyze() async {
```

Comment:
nit: Do you mind calling this something other than analyze? We use that word a lot to refer specifically  to `flutter analyze`. Would something like `verifyCodeSegments` work?

Reply:
Makes sense. I'll rename it to verifyCodeSegments!